### DISCUSSION TITLE
fotokita.grid.id - Tracker Script

### DISCUSSION THREAD

**Discussion by Recehan-Slayer** [realodix/AdBlockID#728] [General]
**Jenis iklan** (hapus yang tidak perlu)
- Lainnya

**Sample URL**
https://fotokita.grid.id/
https://health.grid.id/

**Penjelasan Lebih Lanjut**
`https://atm.sgp1.digitaloceanspaces.com/library/avmLibrary.js`
tracker bukan gan

**Comment by realodix** [comment]
Terlihat seperti file untuk iklan Gan, tapi gak apa-apa ini, karena gak ada yang muncul
